pymaker/__init__.py

```diff
@@ -357,6 +370,34 @@ class TransactStatus(Enum):
      FINISHED = auto()
 
 
+def get_pending_transactions(web3: Web3, address: Address = None) -> list:
+    """Retrieves a list of pending transactions from the mempool."""
+    assert isinstance(web3, Web3)
+    assert isinstance(address, Address) or address is None
+
+    if address is None:
+        address = Address(web3.eth.defaultAccount)
+
+    # Get the list of pending transactions and their details from specified sources
+    if _is_parity(web3):
+        items = web3.manager.request_blocking("parity_pendingTransactions", [])
+        items = filter(lambda item: item['from'].lower() == address.address.lower(), items)
+        items = filter(lambda item: item['blockNumber'] is None, items)
+        txes = map(lambda item: RecoveredTransact(web3=web3, address=address, nonce=int(item['nonce'], 16),
+                                                  latest_tx_hash=item['hash'], current_gas=int(item['gasPrice'], 16)),
+                   items)
+    else:
+        items = web3.manager.request_blocking("eth_getBlockByNumber", ["pending", True])['transactions']
+        items = filter(lambda item: item['from'].lower() == address.address.lower(), items)
+        from pprint import pprint
+        pprint(list(items))
```

Comment:
Overlooked  `pprint`

Reply:
Removing the `pprint` broke the tests.  😅  For some reason, I needed to flatten the iterable created by `filter` before the `map` for it to work properly.  My last commit replaces the `pprint` with a single call to `list(items)`, which works nicely.  Would love to discuss this further.